toast-tk-web-agent/src/main/java/io/toast/tk/agent/ui/i18n/MainAppMessages.java

```diff
@@ -0,0 +1,10 @@
+package io.toast.tk.agent.ui.i18n;
+
+
+public class MainAppMessages {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)